website/docs/api/vocab.mdx

```diff
@@ -93,6 +93,7 @@ given string, you need to look it up in
 > #### Example
 >
 > ```python
+> nlp("I'm eating an apple")
```

Comment:
Maybe we should also add something to the blurb at the beginning of the page? It currently says:

> Note that a `Vocab` instance is not static. It increases in size as texts with
new tokens are processed.

I think it might not be clear that models do not have a vocabulary per sé (perhaps outside the word embeddings if a non-transformer model used).

Reply:
I added 
> Some models may have an empty vocab at initialization.

What do you think?